test/parallel/test-util-internal.js

```diff
@@ -5,50 +5,22 @@ require('../common');
 const assert = require('assert');
 const fixtures = require('../common/fixtures');
 
-const binding = process.binding('util');
-const kArrowMessagePrivateSymbolIndex = binding['arrow_message_private_symbol'];
+const {
+  getHiddenValue,
+  setHiddenValue,
+  arrow_message_private_symbol: kArrowMessagePrivateSymbolIndex
+} = process.binding('util');
 
-function getHiddenValue(obj, index) {
-  return function() {
-    binding.getHiddenValue(obj, index);
-  };
-}
-
-function setHiddenValue(obj, index, val) {
-  return function() {
-    binding.setHiddenValue(obj, index, val);
-  };
-}
-
-const errMessageObj = /obj must be an object/;
-const errMessageIndex = /index must be an uint32/;
-
-assert.throws(getHiddenValue(), errMessageObj);
-assert.throws(getHiddenValue(null, 'foo'), errMessageObj);
-assert.throws(getHiddenValue(undefined, 'foo'), errMessageObj);
-assert.throws(getHiddenValue('bar', 'foo'), errMessageObj);
-assert.throws(getHiddenValue(85, 'foo'), errMessageObj);
-assert.throws(getHiddenValue({}), errMessageIndex);
-assert.throws(getHiddenValue({}, null), errMessageIndex);
-assert.throws(getHiddenValue({}, []), errMessageIndex);
 assert.deepStrictEqual(
```

Comment:
Nit: not sure why this uses `deepStrictEqual`. I think it should be `strictEqual`.